### DISCUSSION TITLE
빌드잇에 있는 에셋을 전부 유니티로 불러올 수는 없나요?

### DISCUSSION THREAD

**Discussion by tooon2602** [naverz/zepeto-studio-kor#1258] [5. Q&A]
유니티에 제페토 에셋을 불러왔더니 테마파크 에셋 밖에 없는데 빌딩 카테고리에있는 한옥 에셋을 사용할 수 없을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 빌드잇에서 원하시는 에셋을 사용해서 월드를 제작하신 후
빌드잇 로더를 사용해서 유니티로 가져오실 수 있습니다 :) 
https://studio.zepeto.me/guides/loading-buildit-maps

**Reply by tooon2602** [reply]
빌드잇 로더로 가져온 에셋을 수정하고 싶은데 방법이 없을까요? 아무리 찾아봐도 모델링 데이터가 없는 것 같습니다.

**Comment by YSHFM** [comment]
모델링일 경우 유니티 내부 패키지 중 fbx exporter가 있습니다. 저같은 경우 모델링을 그렇게 익스포트를 하지만 마테리얼이 따로 익스포트 되진 않았습니다.

**Reply by tooon2602** [reply]
답변 감사합니다. 모델링만 가져다가 UV를 펴던지 해야겠군요..
